syntax = "proto3";

import "csid.proto";

message CSAuthReq
{
	uint32 user_id = 1;
	string token = 2;
}

message SCAuthRsp
{
	enum RESULT
	{
		SUCCESS = 0;
	}
	RESULT result = 1;
}

message CSLoginReq
{
}

message SCLoginRsp
{
	enum RESULT
	{
		SUCCESS = 0;
	}
	RESULT result = 1;
}

message CSLogoutReq
{

}

message CSHeartBeatReq
{

}

message SCHeartBeatRsp
{

}

// pkg
message CSPkgHead
{
	optional CSID id = 1;
}

message CSPkg
{
	CSPkgHead head = 1;
	bytes data = 2;
}
